package.json

```diff
@@ -44,21 +44,21 @@
   "devDependencies": {
     "@babel/core": "^7.6.2",
     "@babel/runtime": "^7.6.2",
-    "@types/jest": "^24.0.18",
-    "babel-jest": "^24.9.0",
+    "@types/jest": "^25.1.4",
```

Comment:
yay!

Reply:
You might need to also upgrade jest-dom in web package to its latest version and change the type on our custom  `isBigNumber` Matcher as the matcher interface for 24 is like `interface Matcher<R>` while 25 has `interface Matcher <R, T>`